[2023-06-15T07:27:47.474+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.format_to_parquet_task manual__2023-06-15T07:20:10.524334+00:00 [queued]>
[2023-06-15T07:27:47.505+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.format_to_parquet_task manual__2023-06-15T07:20:10.524334+00:00 [queued]>
[2023-06-15T07:27:47.509+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-06-15T07:27:47.511+0000] {taskinstance.py:1283} INFO - Starting attempt 2 of 2
[2023-06-15T07:27:47.512+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-06-15T07:27:47.554+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2023-06-15 07:20:10.524334+00:00
[2023-06-15T07:27:47.579+0000] {standard_task_runner.py:55} INFO - Started process 6022 to run task
[2023-06-15T07:27:47.583+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'format_to_parquet_task', 'manual__2023-06-15T07:20:10.524334+00:00', '--job-id', '326', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpewa5fgnz']
[2023-06-15T07:27:47.586+0000] {standard_task_runner.py:83} INFO - Job 326: Subtask format_to_parquet_task
[2023-06-15T07:27:47.704+0000] {task_command.py:388} INFO - Running <TaskInstance: data_ingestion_gcs_dag.format_to_parquet_task manual__2023-06-15T07:20:10.524334+00:00 [running]> on host 658b05ff6f1a
[2023-06-15T07:27:47.833+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-15T07:20:10.524334+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-15T07:20:10.524334+00:00
[2023-06-15T07:27:47.839+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/data_ingestion_gcs_dag.py", line 25, in format_to_parquet
    raise ValueError("No CSV files found. Make sure the dataset has been correctly downloaded and unzipped.")
ValueError: No CSV files found. Make sure the dataset has been correctly downloaded and unzipped.
[2023-06-15T07:27:47.864+0000] {taskinstance.py:1326} INFO - Marking task as FAILED. dag_id=data_ingestion_gcs_dag, task_id=format_to_parquet_task, execution_date=20230615T072010, start_date=20230615T072747, end_date=20230615T072747
[2023-06-15T07:27:47.891+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 326 for task format_to_parquet_task (No CSV files found. Make sure the dataset has been correctly downloaded and unzipped.; 6022)
[2023-06-15T07:27:47.926+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-06-15T07:27:47.970+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
